Dockerfile

```diff
@@ -13,16 +13,17 @@ ENV user app
 
 RUN apk --update add git
 
+RUN adduser -u 9000 -D ${user}
+USER ${user}
+
 ADD . ${app_dir}
 
 WORKDIR ${app_dir}
 
 RUN bundle install --without debugging development
-RUN adduser -u 9000 -D ${user}
-RUN chown -R ${user}:${user} ${app_dir}
```

Comment:
It is needed. I was experimenting with something and forgot to uncomment it. Will push a fix shortly.